django_db_logger/db_log_handler.py

```diff
@@ -27,6 +28,20 @@ def emit(self, record):
             'trace': trace
         }
 
+        if getattr(record, 'request', None) and DJANGO_DB_LOGGER_REQUEST_DETAILS:
+            kwargs['details'] = {}
+            if DJANGO_DB_LOGGER_REQUEST_DETAILS_PATH:
+                kwargs['details']['path'] = getattr(record.request, 'path', None)
+
+            if DJANGO_DB_LOGGER_REQUEST_DETAILS_METHOD:
+                kwargs['details']['method'] = getattr(record.request, 'method', None)
+
+            if DJANGO_DB_LOGGER_REQUEST_DETAILS_GET_PARAMS:
```

Comment:
Can we use `request.query_params` instead and call this `DJANGO_DB_LOGGER_REQUEST_DETAILS_PARAMS` or similar?

Reply:
Unfortunately the request does not necessarily have `query_params` (this is a DRF feature). But I changed the behaviour as discussed: There is only one new config option `` and the new functionality in ``DatabaseLogHandler`` is nested within easily extendable methods.